apps/site/assets/css/_footer.scss

```diff
@@ -493,8 +493,12 @@ address {
 
 .m-footer__policy-links {
   display: block;
-  font-size: $font-size-base-xs;
-  margin-left: 3.5rem;
+  font-size: .9rem;
+
```

Comment:
margin-left: 1rem on mobile

Reply:
![Screenshot 2023-10-24 at 2 44 00 PM](https://github.com/mbta/dotcom/assets/2679229/2aec8509-59e1-48b4-93bd-6af68542869e)
Adding that margin makes the alignment look weird